extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -6,17 +6,17 @@
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
 import * as path from 'path';
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
 import * as yaml from 'js-yaml';
 import * as glob from 'fast-glob';
 import { BookTreeItem, BookTreeItemType } from './bookTreeItem';
 import { maxBookSearchDepth, notebookConfigKey } from '../common/constants';
-import { isEditorTitleFree } from '../common/utils';
 import * as nls from 'vscode-nls';
+import { isEditorTitleFree } from '../common/utils';
 import { promisify } from 'util';
 
 const localize = nls.loadMessageBundle();
-const existsAsync = promisify(fs.exists);
+const existsSync = promisify(fs.exists);
 
```

Comment:
This isn't making it synchronous - all it does is wrap fs.exists in a function that returns a promise. You still need to do something with that promise (await most likely)